src/NuGet.Clients/VisualStudio/VisualStudio.csproj

```diff
@@ -23,6 +23,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>..\..\..\artifacts\VisualStudio\Debug\NuGet.VisualStudio.XML</DocumentationFile>
```

Comment:
Fixed.

This was auto-generated. Actually, this should simply be <DocumentationFile>$(OutputPath)\NuGet.VisualStudio.XML</DocumentationFile> and not relative EnlistmentRoot